operator/src/main/java/org/keycloak/operator/crds/v2alpha1/realmimport/KeycloakRealmImportSpec.java

```diff
@@ -35,6 +42,9 @@ public class KeycloakRealmImportSpec {
     @JsonPropertyDescription("Compute Resources required by Keycloak container. If not specified, the value is inherited from the Keycloak CR.")
     private ResourceRequirements resourceRequirements;
 
+    @JsonPropertyDescription("Optionally set to replace ENV variable placeholders in the realm import.")
```

Comment:
I would not mention env vars. It's an implementational detail hidden from the user (they don't set any env vars).
```suggestion
    @JsonPropertyDescription("Optionally set to replace placeholders in the realm import.")
```

Reply:
The docs https://www.keycloak.org/server/importExport do specifically say the placeholders are resolved from environment variables.